patternfly/patternfly-elements
# Set PFE default accent color to red

* Link(s) to demo pages where this element can be viewed:

---

### What has changed and why
This PR updates orange to red to bring in official RH colors, but also because IE11 doesn't inherit CSS vars so the CTA primary links are orange on prod in IE11. This fixes that.

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. View a page with primary CTAs in IE11. They should be red instead of orange.
2. View the pfe-badge component to ensure colors look the same as they do here https://patternfly.github.io/patternfly-elements/demo/?path=/story/badge--pfe-badge

#### Browser requirements
Your component should work in all of the following environments:


### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
